lua/nvim_lsp/elmls.lua

```diff
@@ -1,59 +1,27 @@
 local skeleton = require 'nvim_lsp/skeleton'
 local util = require 'nvim_lsp/util'
 local lsp = vim.lsp
-local fn = vim.fn
 local api = vim.api
-local need_bins = util.need_bins
-
-local default_capabilities = lsp.protocol.make_client_capabilities()
-default_capabilities.offsetEncoding = {"utf-8", "utf-16"}
 
+local server_name = "elmls"
 local bin_name = "elm-language-server"
 
-local install_dir = util.path.join(fn.stdpath("cache"), "nvim_lsp", "elmls")
-local function get_install_info()
-  local bin_dir = util.path.join(install_dir, "node_modules", ".bin")
-  local bins = { bin_dir = bin_dir; install_dir = install_dir; }
-  bins.elmls = util.path.join(bin_dir, bin_name)
-  bins.elm = util.path.join(bin_dir, "elm")
-  bins.elm_format = util.path.join(bin_dir, "elm-format")
-  bins.elm_test = util.path.join(bin_dir, "elm-test")
-  bins.is_installed = need_bins(bins.elmls, bins.elm, bins.elm_format, bins.elm_test)
-  return bins
-end
-
-local global_commands = {
-  ElmlsInstall = {
-    function()
-      if get_install_info().is_installed then
-        return print("ElmLS is already installed")
-      end
-      skeleton.elmls.install()
-    end;
-    description = 'Install elmls and its dependencies to stdpath("cache")/nvim_lsp/elmls';
-  };
-  ElmlsInstallInfo = {
-    function()
-      local install_info = get_install_info()
-      if not install_info.is_installed then
-        return print("ElmLS is not installed")
-      end
-      print(vim.inspect(install_info))
-    end;
-    description = 'Print installation info for `elmls`';
-  };
-};
-
-util.create_module_commands("elmls", global_commands)
+local installer = util.npm_installer {
+  server_name = server_name;
+  packages = { "elm", "elm-test", "elm-format", "@elm-tooling/elm-language-server" };
+  binaries = {bin_name, "elm", "elm-format", "elm-test"};
+}
 
+local default_capabilities = lsp.protocol.make_client_capabilities()
+default_capabilities.offsetEncoding = {"utf-8", "utf-16"}
```

Comment:
After this PR, `default_capabilities` isn't used anywhere. That might be a mistake. Perhaps `capabilities = default_capabilities` should be added to `default_config`?